scripts/primir_quantification.py

```diff
@@ -138,9 +138,9 @@ def main(args) -> None:
 
     with open(args.bedfile, 'r') as bedfile:
 
-        Fields = namedtuple('Fields',("feat_chr", "feat_start", "feat_end", 
-                                      "feat_id", "feat_score", "strand", "idk", 
-                                      "feat_type", "idkidk", "feat_attributes", 
+        Fields = namedtuple('Fields',("feat_chr", "NA", "feat_type",
+                                      "feat_start", "feat_end", "feat_score", 
+                                      "strand", "NaN", "feat_attributes", 
```

Comment:
I think there's no need for `NA` and `NaN` column names. You can find the column/field meanings in the GFF format description:
https://www.ensembl.org/info/website/upload/gff3.html